cmd/gnofaucet/gnofaucet.go

```diff
@@ -107,16 +112,21 @@ var DefaultServeOptions = serveOptions{
 
 func serveApp(cmd *command.Command, args []string, iopts interface{}) error {
 	opts := iopts.(serveOptions)
+
 	if len(args) != 1 {
 		cmd.ErrPrintfln("Usage: serve <keyname>")
+
 		return errors.New("invalid args")
 	}
+
 	if opts.ChainID == "" {
 		return errors.New("chain-id not specified")
 	}
+
```

Comment:
And this one, maybe the same?

Reply:
Just so you know, I do prefer this new output.

But the previous one was great enough; and I don't want most of the PRs to just become a linting challenge for such minor improvements. At the moment at least.

Reply:
I completely agree with you - adding this `wsl` linter was probably a bit too much, because it enforces _a lot_ of changes on existing codebases, that can cause significant merge conflicts after this PR goes into `master`.

Out of all of the linters added, `wsl` is probably the one that requires the most changes, you can see the vast amount of options it provides [here](https://golangci-lint.run/usage/linters/#wsl)

I've reverted this linter here: